plugins/chunter-resources/src/components/Message.svelte

```diff
@@ -162,47 +162,30 @@
     else AddMessageToSaved(message)
   }
 
-  function openEmojiPalette (ev: Event) {
-    showPopup(
-      EmojiPopup,
-      {},
-      ev.target as HTMLElement,
-      async (emoji) => {
-        if (!emoji) return
-        const me = getCurrentAccount()._id
-
-        const reaction = reactions?.find((r) => r.emoji === emoji && r.createBy === me)
-        if (!reaction) {
-          await client.addCollection(
-            chunter.class.Reaction,
-            message.space,
-            message._id,
-            chunter.class.ChunterMessage,
-            'reactions',
-            {
-              emoji,
-              createBy: me
-            }
-          )
-        } else {
-          await client.removeDoc(chunter.class.Reaction, message.space, reaction._id)
+  async function updateReactions (emoji?: string) {
+    if (!emoji) return
+    const me = getCurrentAccount()._id
+
+    const reaction = reactions?.find((r) => r.emoji === emoji && r.createBy === me)
+    if (!reaction) {
+      await client.addCollection(
+        chunter.class.Reaction,
+        message.space,
+        message._id,
+        chunter.class.ChunterMessage,
+        'reactions',
+        {
+          emoji,
+          createBy: me
         }
-      },
-      () => {}
-    )
+      )
+    } else {
+      await client.removeDoc(chunter.class.Reaction, message.space, reaction._id)
```

Comment:
You should use `client.remove(reaction)` or `client.removeCollection(` because it is `AttachedDoc`

Reply:
Done.